src/test/scala/SentimentAnalysisUtilsTest.scala

```diff
@@ -0,0 +1,15 @@
+import org.scalatest.FunSpec
+import org.scalatest.matchers.ShouldMatchers
+import utils.SentimentAnalysisUtils._
+
+class SentimentAnalysisUtilsTest extends FunSpec with ShouldMatchers{
+  describe("A sentiment analysis utility class") {
+    it ("should detect a negative statement") {
+      detectSentiment("@Sosh_fr iPhone volé, je fais quoi maintenant ?") should equal (NEGATIVE)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Non ascii characters are not allowed](https://www.codacy.com/app/b-fovet/spark-streaming-twitter/file/1663539366/issues/source?bid=2980032&fileBranchId=3034495#l8)